data_steward/resources.py

```diff
@@ -174,6 +175,34 @@ def fields_for(table, sub_path=None):
     return fields
 
 
+def get_table_schema(table_name: str, fields=None) -> list:
```

Comment:
Why is this pulled and copied from https://github.com/all-of-us/curation/blob/develop/data_steward/utils/bq.py#L97.  Why not just use the original code?

Reply:
This is because we are deprecating the functions in utils/bq.py. It will be retired and all functions converted to methods in the new BigQueryClient. This function should ideally be outside the client since it doesn't make any API calls but calling it would require a client to be instantiated, which is why I moved it to resources.py. This way all usages can be converted to this function, and it fits well with fields_for etc.

Reply:
This function is currently defined in three places with this patch.  
1.  It's original location, which is being deprecated.
2. Inside `data_steward/gcloud/bq/__init__.py`, where it has been relocated.
3. By this patch in `resources.py`.

 It's starting to feel like a copy/paste bug.  I know the original will eventually be deleted.  I would still suggest this rightfully belongs in the bq package, because it is returning a list of `SchemaField` objects which are specific to BigQuery.  If all you want is a list of fields, we've got the `fields_for` function in `resources.py`.

Reply:
Yeah the entire bq package will be deleted at some point, but I can use that for now. `fields_for` does not fulfill the requirement, so we cannot use that. Since `get_table_schema` is a static function, I was hoping we don't have to instantiate a client to use it and hence relocated it to `resources.py`. It does use the bigquery library but does't make any API calls.

Reply:
@lrwb-aou I've updated it to use the deprecated function for now.

Reply:
@lrwb-aou I've updated it to use the deprecated function for now.